sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/ReadFeedUdfsTest.java

```diff
@@ -37,12 +37,10 @@ public ReadFeedUdfsTest(CosmosClientBuilder clientBuilder) {
 
     @Test(groups = { "simple" }, timeOut = FEED_TIMEOUT)
     public void readUserDefinedFunctions() throws Exception {
-
-        FeedOptions options = new FeedOptions();
         int maxItemCount = 2;
 
         CosmosPagedFlux<CosmosUserDefinedFunctionProperties> feedObservable = createdCollection.getScripts()
-                                                                                               .readAllUserDefinedFunctions(options);
+                                                                                               .readAllUserDefinedFunctions();
```

Comment:
result validation validates page size, etc. that may fail. you need to update the validation section.

Reply:
Page sizes are controlled by maxItemCount which is no longer part of the FeedOptions since when we start returning PageFlux as results.